.github/workflows/checks.yml

```diff
@@ -42,6 +42,8 @@ jobs:
       - uses: actions/checkout@v3
       - name: Setup
         uses: ./.github/actions/setup
+      - name: Run local fork in background for integration tests
+        run: npx hardhat --tsconfig tsconfig.testing.json node --hostname 127.0.0.1 --fork ${{ secrets.ETHEREUM_RPC_URL }} &
```

Comment:
Not a priority now but worth considering moving to [Anvil](https://book.getfoundry.sh/anvil/#how-to-use-anvil) instead of Hardhat? I think its a good bit faster

Reply:
I moved this from a subtask to a task on asana, so it doesn't block the PR